clients/admin-ui/src/features/dataset/ApproveClassification.tsx

```diff
@@ -43,25 +43,40 @@ const ApproveClassification = () => {
 
     const updatedDataset = getUpdatedDatasetFromClassifyDataset(
       dataset,
-      classifyDataset
+      classifyDataset,
+      classifyCollection?.name
     );
     try {
       const updateResult = await updateDataset(updatedDataset);
       if (isErrorResult(updateResult)) {
         toast(errorToastParams(getErrorMessage(updateResult.error)));
         return;
       }
-
-      const statusResult = await updateClassifyInstance({
-        dataset_fides_key: dataset.fides_key,
-        status: ClassificationStatus.REVIEWED,
+      toast(successToastParams("Collection classified and approved"));
```

Comment:
hm I wonder why this isn't showing up in the video. I tried it locally and it's not showing up there either. I'll look into it a bit more...

Reply:
hm it looks like none of our toasts are working anymore, even on `main` 😱

Reply:
I wonder if it's related to this. I think I ran into this before in the privacy center. Doing what this comment describes fixed my problem. 

[https://github.com/chakra-ui/chakra-ui/issues/5839#issuecomment-1278883908](https://github.com/chakra-ui/chakra-ui/issues/5839#issuecomment-1278883908)

Reply:
heh yeah I remember you fixing this before! I wonder if it came up with the dependabot bump of chakra, or something else...

@TheAndrewJackson want to open a quick PR off of main so we can fix since you've already found the solution? and if there's a release going on, we probably want this in before that...

Reply:
Oof, thank you both! Late Friday night I was losing my mind trying to get the toast messages to work anywhere and resorted to this 🥴  - thanks again for putting some brain power into it

Reply:
@allisonking Yes, I'll take a look at that today. If that's the issue it should be a quick fix 🤞

Reply:
Thank you both for taking this on along with everything else!